scss/_pagination.scss

```diff
@@ -149,15 +154,15 @@
       background-color: var(--#{$prefix}pagination-active-item-bg);
       border-color: var(--#{$prefix}pagination-active-item-border-color);
     }
+
+    &:not([title]) {
```

Comment:
The whole selector might lead to an issue (if the `.page-item` is `.active` as well, it may not happen but if we can anticipate). IMO, the final selector should be `.page-item:first-child .page-link:not([title]), .page-item:last-child .page-link:not([title])`. 
However, I'm fine forcing a11y on this component so the `:not([title])` seems fair to me.